README.md

```diff
@@ -1,2 +1,114 @@
-# Covid-19-Data-Hub
-This is a webapp that display metric data from an API for Covid-19 cases globally. Interactions  Home page  When the page loads, the webapp shows the list of categories that could be filtered out by some parameter. Along with the category name, it displays some numeric values per category that come from the API. When the user clicks (or taps) on a category item, the application navigates to the details page.  Details page  In the details page, the webapp retrieves data from the API to show detailed data in the category. When the user clicks on the "Back" button (&lt;), the user navigates to the home page.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# COVID-19 DATA HUB
+
+> This is a webapp that display metric data from an API for Covid-19 cases globally
+
+![screenshot](./src/images/Home.png)
+![screenshot](./src/images/Details.png)
+![screenshot](./src/images/mobile.png)
+
+Interactions
+
+Home page
+
+When the page loads, the webapp shows the list of categories that could be filtered out by some parameter.
+Along with the category name, it displays some numeric values per category that come from the API.
+When the user clicks (or taps) on a category item, the application navigates to the details page.
+
+Details page
+
+In the details page, the webapp retrieves data from the API to show detailed data in the category.
+When the user clicks on the "Back" button (<), the user navigates to the home page.
+
+## Built With
+
+- Javascript, HTML, CSS
+- React
+- Redux
+- API
+
+## Live Demo
+
+[Live Demo Link](https://cool-bavarois-7aa72a.netlify.app/)
+
+## Project Description
+
+Click here to watch a [video](https://www.loom.com/share/dd48119d67a44859aa058d493145c9e1) that gives a brief description of the project
```

Comment:
- [ ] As requested by the previous reviewer, your video should contain a highlight about a piece of the code you find interesting or are proud of making. Try to open Visual Studio Code (or whatever IDE you use) while you are recording a video and try to present a piece of code that interests you.